work_queue/src/client.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from json_server import WorkQueueServer
+
+def main():
+
+    q = WorkQueueServer()
+
+    #connect to server
+    q.connect('127.0.0.1', 2345)
+  
+    #read in tasks and submit requests
+    f = open("tasks.txt", "r")
```

Comment:
This requires that the task JSON objects be split line-by-line, which will eventually result in some trouble.
Could you make them as native data structures in this source file instead?

Reply:
What exactly do you mean by "native data structures" in this case?

Reply:
Something like this:

```
tasks = [ { task 1 ... }, { task 2 ... }, ... ]

for t in tasks
   submit t
```

Reply:
There is an issue with making the tasks native data structures in the client file. What seems to happen is that putting the JSON description into python as opposed to just treating it as a string in a text file messes things up with the boolean values. Python wants them capitalized, but JSON wants them lower case and so one is going to end up throwing an error.

Reply:
So, the user would have to write `"cache":True` in the python version.
Does `json.dumps(t)` return a string with the proper JSON captialization?